src/plugins/aws/ssh.ts

```diff
@@ -88,6 +88,8 @@ export const awsSshProvider: SshProvider<
 
   preTestAccessPropagationArgs: () => undefined,
 
+  // Starts an SSM session in the terminal by spawning `aws ssm` as a subprocess
+  // Requires `aws ssm` to be installed on the client machine.
```

Comment:
This is not accurate anymore. We aren't spawning an aws ssm subprocess. We are passing aws ssm in a ProxyCommand. I think this comment can be removed.

Reply:
Removed